subsys/nrf_security/Kconfig.psa

```diff
@@ -88,6 +88,11 @@ config PSA_ITS_ENCRYPTED
 	help
 	  Enables authenticated encryption for PSA Internal Trusted Storage files
 
+config PSA_CRYPTO_DRIVER_ALG_PRNG_TEST
+	bool
+	prompt "Application provided PRNG"
```

Comment:
ticket says promptless, but I can't set the option in prj.conf then.

Unsure how this was intended